### DISCUSSION TITLE
Headless Server record start

### DISCUSSION THREAD

**Discussion by BieblRecords** [jamulussoftware/jamulus#1982] [Technical Support & Help]
Can anyone explain in detail how to start a recording on a headless Server? The tutorial didn‘t work or i‘m getting something wrong.
The Service files start but nothing happend.

**Comment by rdica** [comment]
Per https://jamulus.io/wiki/Server-Linux and https://jamulus.io/wiki/Tips-Tricks-More#controlling-recordings-on-linux-headless-servers can you show us logs of what happens when you start a recording?

**Reply by BieblRecords** [reply]
I‘ve got it working. The problem was in the jamulus-headless.service file.
user has to be root or there has to be a user jamulus which doesn‘t exist on my ubuntu server. And i must set ProtectHome=false otherwise no recording is possible.

**Reply by BieblRecords** [reply]
record path is /root/record

**Reply by rdica** [reply]
While I'm happy you got it working, using root for this is bad form. The root account should only be used for administrative functions. I strongly recommend using an existing user or creating one, and use that users home directory to write files to.

**Reply by BieblRecords** [reply]
Hi rdica,
many thanks for your answers so far.
I removed everything on my server, created a new user with sudo rights:
     sudo adduser user
     sudo usermod -aG sudo user
     made a complete new jamulus-headless install
     and removed ProtectHome=false in the jamulus-headless.service file

Is this the right way?

Regards

**Reply by rdica** [reply]
Sure, looks fine. The user doesn't need sudo rights unless you plan to use that same user for other tasks besides running jamulus.
Does the service file have a
`User=blah`
line, and properly set to your new user?

**Reply by BieblRecords** [reply]
Hi rdica again,
i´ve set the user to my_own_user.
I made sudo deluser my_own_user sudo to remove my_own_user from group sudoers.

As user root i can control the jamulus-headless.service:
     sudo service jamulusToggleRec start 
     sudo service jamulus-headless start
     sudo service jamulus-headless stop
     sudo service jamulusToggleRec start
     sudo service jamulus-headless status

As user my_own_user i can only check the status of the service:
     service jamulus-headless status

Regards

**Reply by rdica** [reply]
root doesnt need to use sudo, a user would need it to exec those sudo commands you listed above.

**Reply by BieblRecords** [reply]
👍 You‘re absolutely right ;). Many  thanks for your support.
